storage/storage.go

```diff
@@ -36,12 +35,16 @@ var LicenseHandler Handler
 
 func Connect() {
 	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
-	MongoClient, err := mongo.Connect(ctx, options.Client().ApplyURI(config.Global.MongoURL))
+	MongoClient, err := mongo.Connect(ctx, options.Client().ApplyURI(config.Global.DBOptions.Mongo.URL))
```

Comment:
Why should we connect mongo if user sets `file`? I think this should be inside if-else.